docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -0,0 +1,397 @@
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<allocation>``                  | Configuration regarding allocation behavior.     | |PartAlloc|         |         |
+|                                   | |br| It expects a |br| |PartAlloc|.              |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+
+.. _Port:
+
+Port Configuration
+""""""""""""""""""
+
+According to the `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ (Section 9.6.1.1), the
+|RTPSParticipants|'discovery traffic unicast listening ports are calculated using the following equation:
+:math:`7400 + 250 * DomainId + 10 + 2 * ParticipantId`.
+Therefore the following parameters can be specified:
+
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| Name                         | Description                                          | Values          | Default      |
++==============================+======================================================+=================+==============+
+| ``<portBase>``               | Base ``port``.                                       | ``uint16``      | 7400         |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<domainIDGain>``           | Gain in |DomainId|.                                  | ``uint16``      | 250          |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<participantIDGain>``      | Gain in |DomainParticipant_Id|.                      | ``uint16``      | 2            |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<offsetd0>``               | Multicast metadata offset.                           | ``uint16``      | 0            |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<offsetd1>``               | Unicast metadata offset.                             | ``uint16``      | 10           |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<offsetd2>``               | Multicast user data offset.                          | ``uint16``      | 1            |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+| ``<offsetd3>``               | Unicast user data offset.                            | ``uint16``      | 11           |
++------------------------------+------------------------------------------------------+-----------------+--------------+
+
```

Comment:
Maybe a warning saying that changing this will break compatibility with other RTPS compliant implementations, or even with another app using fast with default ports config